src/profiling/symbolizer/BUILD.gn

```diff
@@ -58,6 +65,50 @@ if (enable_perfetto_trace_processor) {
   }
 }
 
+if (enable_perfetto_llvm_symbolizer) {
+  _llvm_flags_output =
+      exec_script("../../../tools/get_llvm_flags.py", [], "scope")
```

Comment:
do we really need all those flags? are they realistically ever going to change?
IMHO it's more pragmatic to bake the flags in here, and change them if and whenever they chagne to catch up.
Unless there is a good reason why those flags legitimately need to be polled at runtime
I know about pkgconfig.... but I also know in practice in most cases those flags never change. I'd not add this level of complexity unless there is a strong case why it's needed.
Can't we use the paths we get from the existing linux_find_lvvm.py?